pkg/apis/alicloud/types_infrastructure.go

```diff
@@ -94,6 +94,14 @@ type Zone struct {
 	Worker string
 	// Workers specifies the worker CIDR to use.
 	Workers string
+	// NatGatewayConfig specifies the eips to bind
+	// +optional
+	NatGateway *NatGatewayConfig
+}
+
+// NatGatewayConfig contains specified eips to bind nat gateway
+type NatGatewayConfig struct {
+	IPAddresses []string
```

Comment:
Doc string missing

Also, see above comment: Why taking a list here if you only use the first one?